.github/workflows/mac.yml

```diff
@@ -0,0 +1,58 @@
+name: MacOS Build
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+jobs:
+  build:
+    runs-on: macOS-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+      with:
+        # Needed for commands that depend on git tags
+        fetch-depth: 0
+        ref: ${{ github.event.pull_request.head.sha }}
+    - name: Setup Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: '1.15.5'
+    - name: Cache go modules
+      id: cache-go-mod
+      uses: actions/cache@v2.1.5
+      with:
+        path: ~/go/pkg/mod
+        key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+        restore-keys: |
+          ${{ runner.os }}-go-
+    - name: Cache ffmpeg
+      id: cache-ffmpeg
+      uses: actions/cache@v2.1.5
+      with:
+        path: ~/compiled
+        key: ${{ runner.os }}-ffmpeg-${{ hashFiles('**/install_ffmpeg.sh') }}
+        restore-keys: |
+          ${{ runner.os }}-ffmpeg
+    - name: Install dependencies
+      run: brew install coreutils
+    - name: Install go modules
+      if: steps.cache-go-mod.outputs.cache-hit != 'true'
+      run: go mod download
+    - name: Install ffmpeg
+      if: steps.cache-ffmpeg.outputs.cache-hit != 'true'
+      run: ./install_ffmpeg.sh
+    - name: Build binaries
+      run: |
+        export PKG_CONFIG_PATH=~/compiled/lib/pkgconfig
+        ./ci_env.sh make livepeer livepeer_cli livepeer_bench livepeer_router
```

Comment:
Is there like a `make all` (or perhaps just `make`)?

Reply:
Just `make` would indeed be simpler!

There is, but previously it wouldn't build the livepeer_bench and livepeer_router binaries mentioned here.

bade226 updates the Makefile so `make` will also build livepeer_bench and livepeer_router
b337997 simplifies the GH actions to all just run `make`